src/cmds/init.py

```diff
@@ -44,6 +44,14 @@ def migrate_from(old_root, new_root, manifest):
     # and make what may have been relative paths absolute
 
     plots_config = load_config(new_root, "plots.yaml")
+
+    new_plots_root = path_from_root(
+        old_root,
```

Comment:
`new_plots_root` should be named `old_plots_root` and should be based off of `config.yaml["harvester"].get("plot_root", ".")`.

Reply:
Why "." and not "plots"? The default place for plots was "./plots".